<ui version="4.0" >
 <class>jewelryPurchasingAddClass</class>
 <widget class="QWidget" name="jewelryPurchasingAddClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>采购信息编辑</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QWidget" native="1" name="widget_2" >
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>供货商名称：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="cbSupplierName" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="pbQuerySupplier" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>商品名称：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="cbCommodityName" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="pbCommodityName" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>仓库名称：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="cbWarehouseName" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QPushButton" name="pbQueryWarehouse" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>采购员</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QComboBox" name="cbPurchaseName" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QPushButton" name="pbQueryStaff" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>商品数量</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QSpinBox" name="spCommodityCount" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="maximum" >
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>商品单价</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QDoubleSpinBox" name="dbCommodityPrice" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="decimals" >
         <number>4</number>
        </property>
        <property name="maximum" >
         <double>999.990000000000009</double>
        </property>
        <property name="value" >
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_10" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>备注：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2" >
       <widget class="QTextEdit" name="textEdit" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>340</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbOK" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>确定</string>
        </property>
        <property name="autoDefault" >
         <bool>true</bool>
        </property>
        <property name="default" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCancel" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
